src/app/dashboard/store/actions/projects.ts

```diff
@@ -94,5 +94,5 @@ export class ReleaseProjectAttemptAction implements Action {
 export const RELEASE_PROJECT_SUCCESS = 'RELEASE_PROJECT_SUCCESS';
 export class ReleaseProjectSuccessAction implements Action {
     readonly type = RELEASE_PROJECT_SUCCESS;
-    constructor(public project: Partial<Project>) {}
+    constructor(public release: Partial<Project>, public projectId: string) {}
```

Comment:
I am not sure about : ```public release: Partial<Project>``` you are creating a project but you call it release? Is wrong the name or the type?

Reply:
also here should be :  ```constructor(public release: Partial< Release >, public projectId: string) {}``` ?